core/src/components/tab-bar/tab-bar.tsx

```diff
@@ -1,18 +1,20 @@
 import type { ComponentInterface, EventEmitter } from '@stencil/core';
 import { Component, Element, Event, Host, Prop, State, Watch, h } from '@stencil/core';
 
-import { getIonMode } from '../../global/ionic-global';
+import { getIonStylesheet } from '../../global/ionic-global';
 import type { Color, TabBarChangedEventDetail } from '../../interface';
 import type { KeyboardController } from '../../utils/keyboard/keyboard-controller';
 import { createKeyboardController } from '../../utils/keyboard/keyboard-controller';
 import { createColorClasses } from '../../utils/theme';
 
 /**
+ * @virtualProp {true | false} useBase - useBase determines if base components is enabled.
  * @virtualProp {"ios" | "md"} mode - The mode determines which platform styles to use.
  */
 @Component({
   tag: 'ion-tab-bar',
   styleUrls: {
+    base: 'tab-bar.scss',
```

Comment:
This one vanishes completely with `use-base` on and I feel like it shouldn't 😆

Reply:
TabBar expects to have a height set by iOS/MD modes. I added a basic height of 50px just so its shows. LMK if you think that should be different.